src/upload-artifact.ts

```diff
@@ -43,7 +43,7 @@ async function run(): Promise<void> {
       // If s3Prefix is left blank then just use the actual default derived from the github context
       const s3Prefix =
         inputs.s3Prefix === ''
-          ? `${github.context.repo.owner}/${github.context.repo.repo}/${github.context.runId}/${inputs.artifactName}`
+          ? `${github.context.repo.owner}/${github.context.repo.repo}/${github.context.runId}/${github.context.runNumber}/${inputs.artifactName}`
```

Comment:
```suggestion
          ? `${github.context.repo.owner}/${github.context.repo.repo}/${github.context.runId}/${github.context.runAttempt}/${inputs.artifactName}`
```

Reply:
runAttempt doesn't actually exist

Reply:
This is my PR to actually add it: https://github.com/actions/toolkit/pull/1027